k8s/installation/observability.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: Monitoring the Kubernetes Operator
+sidebarTitle: Observability
+description: Learn how to enable and customize ngrok's Kubernetes ingress Observability features.
+---
+
+Configuring metrics and logging for the ngrok Kubernetes Operator is essential for gaining visibility into how your ngrok ingress resources behave within your cluster.
```

Comment:
> how your ngrok ingress resources behave within your cluster

This part feels a bit off to me. The "Ingress resource" is basically like a json API payload. Its more "how the operator is handling your ingress resources and other crds".